### PR TITLE
java: catch exceptions if we fail to load mpi_java

### PR DESCRIPTION
**Pull request by hppritcha**
library.

Signed-off-by: Howard Pritchard <howardp@lanl.gov>

### CODE REVIEW

**Comment by hppritcha** [comment]
a step along the way to retaining ease of use with prrte for java apps

File: ompi/mpi/java/java/MPI.java [hunk 1]
```diff
@@ -204,7 +204,15 @@
...
 	static
 	{
-		System.loadLibrary("mpi_java");
+		try
+                {
+                    System.loadLibrary("mpi_java") ;
+                }
+                catch (UnsatisfiedLinkError e)
+                {
+                    System.err.println("mpi java lib failed to load" + e  + "\n") ;
```
Comment (kawashima-fj) [inline ompi/mpi/java/java/MPI.java:213]: Please insert a space of something before the Java exception message.

```suggestion
                    System.err.println("mpi java lib failed to load: " + e  + "\n") ;
```

**Review by kawashima-fj — CHANGES_REQUESTED** [review]
@hppritcha Just a question. How the behavior is changed? Only the messsage change? Exiting cleanly?

**Comment by hppritcha** [comment]
@kawashima-fj check now, hopefully i got the tabs right!

File: ompi/mpi/java/java/MPI.java [hunk 2]
```diff
@@ -204,7 +204,15 @@
 
 	static
 	{
-		System.loadLibrary("mpi_java");
+		try
+		{
+			System.loadLibrary("mpi_java") ;
+                }
```
Comment (kawashima-fj) [inline ompi/mpi/java/java/MPI.java:210]: This line still has space-indentation.

Comment (hppritcha) [reply ompi/mpi/java/java/MPI.java]: okay i think that fixed it.

**Comment by hppritcha** [comment]
@kawashima-fj  i think line 210 is fixed now

**Review by kawashima-fj — APPROVED** [review]

**Pull request merged**